app.rb

```diff
@@ -6,6 +6,10 @@
 require "dotenv"
 require "text"
 require "sanitize"
+require "pycall"
+
+# TODO: figure out how to install ftfy in the context of this project/deployment, see https://github.com/rspeer/python-ftfy
+ftfy = PyCall.import_module("ftfy")
```

Comment:
@mduleone, this is deployed on [Heroku](https://dashboard.heroku.com/apps/careof-trebekbot), [right](https://careof.slack.com/archives/C014TT957N3/p1654174800134229)?

I'll probably need to tweak some things to get pycall working properly, as described in https://github.com/mrkn/pycall.rb/pull/156/files, as well as to get the python runtime and ftfy package installed.

Reply:
I think the command to run will be `heroku buildpacks:add --index 1 https://github.com/dsounded/heroku-buildpack-python.git#43cf2dec7254605e894a026e60ef214f5d115168`

Reply:
([done](https://careof.slack.com/archives/C014TT957N3/p1661365585959199?thread_ts=1661356586.226349&cid=C014TT957N3))